example/src/partials/head.js

```diff
@@ -1,6 +1,7 @@
 import styles from './head.css'
 import x from 'test/test.entry'
 import y from 'test/test.css'
+import stylesheet from '@kaliber/build/lib/stylesheet'
```

Comment:
Can't we export this stylesheet from the module?
```js
import { stylesheet } from '@kaliber/build'

Reply:
We theoretically could, but that would cause other issues. If you look at the changes in `build.js` you see we have added patterns in both the `externals` and the loader. These exceptions allow us to treat this code differently. If we change that pattern, all of `@kaliber/build` will stop to be external.